molgenis-core-ui/src/main/javascript/modules/react-components/wrapper/Ace.js

```diff
@@ -34,7 +34,7 @@
 			};
 		},
 		getInitialState: function() {
-			return {value: this.props.value};
+			return {value: this.props.value || ''};
```

Comment:
This was a bug fix for a bug that we triggered by importing a higher version of ace than the one the master used. We went back to the lower version so this can be removed. Well spotted.